content/core/core_news/2020-11-03-8-7-0-available.md

```diff
@@ -19,8 +19,8 @@ Faster sorting by field. When a doc-value field is also indexed with points, Luc
 
 Faster flushing of stored fields when index sorting is enabled. This can significantly speed up indexing when a non-negligible amount of data is stored in the index and index sorting is enabled.
 
-Further details of changes are available in the change log available at: http://lucene.apache.org/core/8_7/changes/Changes.html
+Further details of changes are available in the change log available at: http://lucene.apache.org/core/8_7_0/changes/Changes.html
 
 Please report any feedback to the mailing lists (http://lucene.apache.org/core/discussion.html)
 
-Note: The Apache Software Foundation uses an extensive mirroring network for distributing releases. It is possible that the mirror you are using may not have replicated the release yet. If that is the case, please try another mirror. This also applies to Maven access.
\ No newline at end of file
+Note: The Apache Software Foundation uses an extensive mirroring network for distributing releases. It is possible that the mirror you are using may not have replicated the release yet. If that is the case, please try another mirror. This also applies to Maven access.
```

Comment:
Unintended change?

Reply:
Yes. I made this PR on browser. I don't know why it has changed...